### PR TITLE
Module1/nationality

### PR DESCRIPTION
**Pull request by Marin-Andres**

### CODE REVIEW

File: App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Profile/Location.jsx [hunk 1]
```diff
@@ -196,12 +200,41 @@ export class Address extends React.Component {
...
+            this.props.saveProfileData(newData);
+        }
+        else {
+            TalentUtil.notification.show("Invalid nationality", "error", null, null);
+        }
     }
 
     
     render() {
+        const countries = Object.keys(Countries).map(c => ({value: c, title: c}));
```
Comment (kestermvp) [inline App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Profile/Location.jsx]: What if accidentally, the backend returns no **Countries** or it becomes a string or boolean. This condition will then cause an error, or will crash the app.

What you can do here is to verify first if **Countries** is a valid object or array then do the mapping

**Review by kestermvp — CHANGES_REQUESTED** [review]
Please see my notes, and on my firs try, after I updated my nationality and reload the page, there is nothing showing unless I updated again my Nationality.

File: App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Profile/Location.jsx (hunk 1 above)
Comment (Marin-Andres) [reply App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Profile/Location.jsx]: Thanks Kester, I've added a new layer of safety to make sure countries will return an empty array if Countries is invalid.

**Comment by Marin-Andres** [comment]
I couldn't quite reproduce the behavior you described even with a new profile. I did add a refresh after updating profile data that I think should take care of this responsiveness on this component and others

**Pull request merged**